internal/cmd/schema-registry/command_schema_create.go

```diff
@@ -113,3 +148,26 @@ func (c *command) schemaCreate(cmd *cobra.Command, _ []string) error {
 	output.Printf(errors.RegisteredSchemaMsg, response.Id)
 	return nil
 }
+
+func readMetadataAndRuleset(metadataPath string, metadata *srsdk.Metadata, rulesetPath string, ruleset *srsdk.RuleSet) error {
```

Comment:
We should make a general purpose function with only two parameters here. It would look like `func read(path string, v any)`

Reply:
Also, you should really be using `json.NewDecoder().Decode()` here!